src/main.rs

```diff
@@ -60,11 +41,48 @@ async fn main() {
             }
         }
 
-        if get_time() - last_updated > UPDATE_TIMER {
+        draw_text(
+            "Start/Stop game with SPACE",
+            10.,
+            screen_height() - CELL_SIZE * 2.5,
+            32.,
+            RED,
+        );
+
+        draw_text(
+            "Click on the cells to create or delete one",
+            10.,
+            screen_height() - CELL_SIZE,
+            32.,
+            RED,
+        );
+
+        if is_mouse_button_pressed(MouseButton::Left) {
+            let (mouse_x, mouse_y) = mouse_position();
+            let mark_cell: (usize, usize) = (
+                (mouse_x / CELL_SIZE).floor() as usize,
+                (mouse_y / CELL_SIZE).floor() as usize,
+            );
+            draw_rectangle(
+                mark_cell.0 as f32 * CELL_SIZE,
+                mark_cell.1 as f32 * CELL_SIZE,
```

Comment:
I would use `let (mark_cell_x, mark_cell_y) = (` here to avoid indexing and also not repeat `usize` since you are already using `as usize` below.